dapp/src/components/Sidebar.tsx

```diff
@@ -1,117 +0,0 @@
-import React, { ComponentProps, useEffect } from "react"
-import {
-  Box,
-  Card,
-  CardBody,
-  Flex,
-  useMultiStyleConfig,
-  Image,
-} from "@chakra-ui/react"
-import { useScrollbarVisibility, useSidebar } from "#/hooks"
```

Comment:
I think we can remove the `useScrollbarVisibility` hook. 🤔

Reply:
Ref commit:  9a506e56